R/internal.R

```diff
@@ -846,7 +847,7 @@ outputmatrix <- function(m, single.row, neginfstring, posinfstring,
 format.column <- function(x, single.row = FALSE, digits = 2) {
     
     #max length before first dot and max length of parentheses
-    dots <- gregexpr("\\.", x)
+    dots <- lapply(x, nchar)
```

Comment:
I'm not sure what this change here does. But generally, it strikes me as odd to wrap `nchar` in an `apply` command because `nchar` can accept a vector of characters as input.

Then in line 855 (and possibly other lines), you will have to remove `[[i]]` because the output will be a vector and not a list.

Reply:
I replaced with 
`dots <- nchar(as.vector(x))`
this should be safer